### ISSUE TITLE
Make supporting links tomato color instead of cyan

### ISSUE THREAD

**Issue by zackkrida** [creativecommons/cccatalog-frontend#1221]
The relevancy yes/no and safer browsing links should be switched to the `tomato` brand color for accessibility reasons. The cyan has insufficient contrast on the gray background.

<img width="1169" alt="Screen Shot 2020-09-23 at 6 44 19 AM" src="https://user-images.githubusercontent.com/6351754/94043373-8c454f00-fd68-11ea-92ea-bd8478fcf184.png">

**Comment by ritiktaneja** [comment]
I would like to work on this issue.

**Comment by zackkrida** [comment]
@ritiktaneja Great!

**Comment by Yashs911** [comment]
Hey, @zackkrida I would like to work on this.

**Comment by zackkrida** [comment]
Sure! Anyone who wants to work on it can 😺

**Comment by Yashs911** [comment]
@zackkrida  Can you please provide the initial push as I am new to open source. In which file do I need to make the changes I tried searching but the codebase is too large.

**Comment by kalyani489** [comment]
Can I work on this?

**Comment by Yashs911** [comment]
@zackkrida I opened the website and did the changes using Inspect. But I am unable to find the file in the codebase as it is too large and I am new to open source. So please guide in which file do I need to make the changes.
